### PR TITLE
`registry-cli` feat: add `restart` and `start` commands

### PR DESCRIPTION
**Pull request by gamemaker1**

### CODE REVIEW

File: tools/cli/src/toolbox/registry/restart.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		.exec('docker compose up --force-recreate -d')
+		.catch((error: Error) => {
+			events.emit('registry.create', {
+				status: 'error',
+				operation: 'restart-registry',
+				message: `An unexpected error occurred while restarting the registry: ${error.message}`,
+			})
+		})
+	// Wait for 40 seconds for them to start
+	await new Promise((resolve) => setTimeout(resolve, 40000))
```
Comment (dileepbapat) [inline tools/cli/src/toolbox/registry/restart.ts:26]: can poll and check status?

File: tools/cli/src/toolbox/registry/restart.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	const { events, system } = toolbox
+
+	events.emit('registry.restart', {
+		status: 'progress',
+		operation: 'restart-registry',
+		message: 'Restarting all registry containers',
+	})
+
+	await system
+		.exec('docker compose up --force-recreate -d')
```
Comment (dileepbapat) [inline tools/cli/src/toolbox/registry/restart.ts:17]: force recreate will destroy container, may not be expected behavior

File: tools/cli/src/toolbox/registry/restart.ts (hunk 1 above)
Comment (gamemaker1) [reply tools/cli/src/toolbox/registry/restart.ts:26]: Sure, will do

File: tools/cli/src/toolbox/registry/restart.ts (hunk 2 above)
Comment (gamemaker1) [reply tools/cli/src/toolbox/registry/restart.ts:17]: I use `docker compose up -d --force-recreate` to restart the containers after configuring keycloak and regenerating the  client secret in the `registry init` command as well. It ensures that the registry container picks up on the new client secret or any other configuration changes and doesn't cause a loss of configuration in the keycloak container either. 

On the other hand, `docker compose restart` did not get the registry to pick up the new/changed configuration, and that is the main purpose of the `registry restart` command.

File: tools/cli/src/toolbox/registry/restart.ts (hunk 1 above)
Comment (gamemaker1) [reply tools/cli/src/toolbox/registry/restart.ts]: Done!

File: tools/cli/src/toolbox/registry/restart.ts (hunk 2 above)
Comment (gamemaker1) [reply tools/cli/src/toolbox/registry/restart.ts]: I have also added a `--soft` option to enable restarting without destroying the container.

**Comment by gamemaker1** [comment]
@dileepbapat Could you please review and merge this? Thanks!

**Pull request merged**